ui/src/app/applications/components/applications-list/applications-table.tsx

```diff
@@ -59,7 +66,16 @@ export const ApplicationsTable = (props: {
                 applications-list__entry applications-list__entry--health-${app.status.health.status} ${selectedApp === i ? 'applications-tiles__selected' : ''}`}>
                                         <div
                                             className={`row applications-list__table-row`}
-                                            onClick={e => ctx.navigation.goto(`applications/${app.metadata.namespace}/${app.metadata.name}`, {}, {event: e})}>
+                                            // Use not namespaced endpoint if namespace is undefined
+                                            onClick={e =>
+                                                ctx.navigation.goto(
+                                                    typeof app.metadata.namespace !== 'undefined'
+                                                        ? `/applications/${app.metadata.namespace}/${app.metadata.name}`
```

Comment:
i see these links in two places, so it's better to move this logic into dedicated function